chains/evm/scripts/compile_all

```diff
@@ -10,6 +10,7 @@ echo " └───────────────────────
 # as specified in the foundry.toml.
 OPTIMIZE_RUNS_OFFRAMP=800
 OPTIMIZE_RUNS_FEE_QUOTER=8000
+OPTIMIZE_RUNS_BURN_MINT_FAST_TRASNFER_TOKEN_POOL=8300
```

Comment:
Please just make this 8k so we don't have custom params for every contract. FeeQuoter and the pool can share 8k